### PR TITLE
Rerender 1.3.2

### PR DESCRIPTION
**Pull request by bollwyvl**
Whoops! This is a redo of #2 from the correct repo!
- conda smith 1.3.2
- upstream 2.70.0.51

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,13 +4,9 @@
...
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```
Comment (jakirkham) [inline .travis.yml]: So this re-rendering seems off. We have seen this issue before<sup>1</sup>. The fix was to get everything from the conda-forge channel in `root`. Maybe try that either in your current Miniconda install or a new one and see if that doesn't work.


1 - https://github.com/conda-forge/conda-smithy/issues/223

**Comment by jakirkham** [comment]
No worries. Thanks for doing this. There are some re-rendering issues that seem linked to build failures to. Please try the proposed fix. Let's see if that doesn't fix them.

File: .travis.yml (hunk 1 above)
Comment (bollwyvl) [inline .travis.yml]: i'll try a new install... got some meetings, so it will likely be this evening!

**Comment by bollwyvl** [comment]
I tried with a brand-spanking-new Miniconda3, and `rerender` didn't result in any change over the previous HEAD of this branch...

**Comment by jakirkham** [comment]
Should we close this out in favor of PR ( https://github.com/conda-forge/newrelic-feedstock/pull/4 )?

**Comment by jakirkham** [comment]
Should we close this out in favor of PR ( https://github.com/conda-forge/newrelic-feedstock/pull/4 )?

**Comment by bollwyvl** [comment]
> @jakirkham Should we close this out in favor of PR ( #4 )?

Yes! Never figured out what was wrong here!

**Pull request closed without merging**